src/pages/documentation/places/places-to-aep.md

```diff
@@ -0,0 +1,73 @@
+# Steps to configure Places Event forwarding for Experience Platform
```

Comment:
Places Service event forwarding to Adobe Experience Platform

Reply:
You need to also update the gatsy-config file to include this file in the left nav menu